lib/looker-sdk/client.rb

```diff
@@ -58,7 +58,9 @@ def initialize(opts = {})
       @original_options = options.dup
 
       load_credentials_from_netrc unless application_credentials?
-      load_swagger
+      if !@lazy_swagger
+        load_swagger
+      end
```

Comment:
This mainly benefits unittests (to avoid always mocking out loading the swagger when we want to test unrelated functionality). But others might have a reason to delay swagger loading too?

I generally don't like modifying production code for the needs of tests but this seemed like a reasonable change to avoid excessive test mocking.

Reply:
Avoiding mocking is a great way to get my approval